mappings/net/minecraft/ChatMessageSender.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/class_7436 net/minecraft/ChatMessageSender
+	FIELD comp_756 uuid Ljava/util/UUID;
+	FIELD comp_757 name Lnet/minecraft/class_2561;
```

Comment:
These and the getters don't need to be explicitly named when name proposal infers them.